src/plugins/data/public/ui/filter_bar/filter_editor/index.tsx

```diff
@@ -294,7 +295,6 @@ class FilterEditorUI extends Component<Props, State> {
           onChange={this.onFieldChange}
           singleSelection={{ asPlainText: true }}
           isClearable={false}
-          className="globalFilterEditor__fieldInput"
```

Comment:
If you remove this class then the change you made to #355 is not going to work?

Reply:
Yes, initially I used the subjects, now that I use the class, I Will revert this change in this PR